libs/blocks/table/table.css

```diff
@@ -561,7 +540,7 @@ header.global-navigation {
     display: none;
   }
 
-  .table:not(.merch) .row-highlight:has(:only-child) {
+  .table:not(.merch) .row-highlight:has(> :only-child) {
```

Comment:
Any specific reasoning to add the child combinator here?

Reply:
Yes, adding additional styling, such as bold/emphasis/etc. to highlight text wraps it in the respective tag (`strong`, `em`, etc.), making each such text an only child. I'm not 100% sure what the original intention of this was, but I've restricted it to its probable initial purpose.

Reply:
Thanks for the context!